Trends docs refresh (#8420)
PostHog/posthog.com: contents/docs/product-analytics/trends/tips.mdx

<ProductScreenshot
    imageLight = {personsLight} 
    imageDark = {personsDark}
    classes="rounded"
    alt="All persons who performed an event"
/>

From this menu, you can:
- Watch specific session replays where the user performed the selected event
- Dive deep into the properties of each individual user
- Save the entire list of users as a cohort
- Export the list of users to a CSV for further analysis

## Creating insights in notebooks

You can add all of PostHog's insight types natively in [Notebooks](/docs/notebooks). These include trends, funnels, retention, paths, stickiness, lifecycle, and SQL.

Each of them can be completely customized within notebooks like you would in insights. To do this, click the three-line reverse pyramid icon.

<ProductVideo
    videoLight={insightsLight} 
    videoDark={insightsDark}
    alt="Insight" 
    classes="rounded"
/>

You can also add insights from the insights list by clicking, holding, and dragging the title onto the right-side popover.

## Add units / custom prefixes and postfixes

You can change the unit of any trends insights, or add custom prefixes/postfixes - e.g. with a prefix of `$`, 123.45 will be displayed as "$123.45".

> Units include duration (seconds or miliseconds) and pertentage.

Reviewer: ```suggestion
Units include duration (seconds or milliseconds) and percentage.
```

<ProductScreenshot
    imageLight = {prefixLight} 
    imageDark = {prefixDark}
    classes="rounded"
    alt="Unit and prefix options"
/>

> To do so, click on 'Options' dropdown next to the chart type and select you want.

Reviewer: ```suggestion
To do so, click on 'Options' dropdown next to the chart type and select the one you want.
```

<ProductScreenshot
    imageLight = {legendLight} 
    imageDark = {legendDark}
    classes="rounded"
    alt="Series legend"
/>

Also under the 'Options' dropdown is the ability to enable values on trends, and the legend, like in the screenshot above.

## View source (Beta)

> Insight filters follow a declarative format that you can view and edit. Open the insight editor by clicking on the `{}` icon next to the save insight button and you will see the source for the current insight configuration. Using this editor you can copy-and-paste complete or partial insights and understand what is going on under the hood. We admit this is a bit of a power user feature, but if you're so inclined you might even find it easier to edit insights this way.

Reviewer: ```suggestion
Insight filters follow a declarative format that you can view and edit. To do so, open the insight editor by clicking on the `{}` icon next to the save insight button and you will see the source for the current insight configuration. 

Using this editor you can copy-and-paste complete or partial insights and understand what is going on under the hood. We admit this is a bit of a power user feature, but if you're so inclined you might even find it easier to edit insights this way.
```
